src/fetchers/workspace-fetcher.js

```diff
@@ -39,8 +39,7 @@ export default class WorkspaceFetcher {
       dest: this.dest,
       package: {
         ...pkg,
-        _uid: '',
-        version: '0.0.0',
+        _uid: (pkg && pkg.version) || '',
```

Comment:
`pkg.version` should be enough - workspaces without versions are invalid and [ignored](https://github.com/yarnpkg/yarn/blob/master/src/config.js#L632-L635)